api/swagger.yaml

```diff
@@ -1387,6 +1387,7 @@ definitions:
           Aliases:
             - "server_x"
             - "server_y"
+        database_nw: {}
```

Comment:
Is this a valid example? I see that `verifyNetworkingConfig()` would possibly invalidate it?

Reply:
I don't think so, `verifyNetworkingConfig()` checks if this is a nil value. `{}` would be unmarshaled into an `EndpointSettings` with no fields set.